Twig/Extension/MediaExtension.php

```diff
@@ -192,6 +192,16 @@ public function getMediaService()
     }
 
     /**
+     * NEXT_MAJOR: Remove this method when bumping requirements to twig 2.
+     *
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
This should not be needed @greg0ire \Twig_Extension already contains this method on 1.x